jupyter-notebooks/analytics/user-guide/2. Analytic Feeds Results.ipynb

```diff
@@ -68,14 +84,18 @@
     "| Vessel Detection | Object Detection | Detection Features (Polygons) | WFS 3 (Detection Features) |\n",
     "| Building Detection | Segmentation (Classification) | Raster Mask / Basemaps | WFS 3 (Quads) / XYZ Tiles |\n",
     "| Road Detection | Segmentation (Classification) | Raster Mask / Basemaps | WFS 3 (Quads) / XYZ Tiles |\n",
+    "| Building Construction Detection | Segmentation (Classification) | Raster Mask / Basemaps | WFS 3 (Quads) / XYZ Tiles |\n",
```

Comment:
above you say feed types are: `Object Detection`, `Segmentation`, and `Change Detection`.

but here you have different looking feed types: `Vessel Detection`, `Building Detection`, `Road Detection`, `Building Construction Detection`, `Road Construction Detection`

Reply:
Building Construction Detection and Road Construction Detection are both just two different types of Change Detection. Just as for Segmentation we have Road Segmentation and Building Segmentation.

Reply:
I've updated the table a bit.